src/index.html

```diff
@@ -1,6 +1,18 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
```

Comment:
The indentation for attributes should be 2 spaces. The attributes `href` and `rel` are indented with 4 spaces instead of 2.